src/styles/blocks/container.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  --columm-count: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--columm-count), 200px);
+  gap: 46px 48px;
+  place-content: center;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    --columm-count: 2;
+  }
+
+  @media (min-width: 768px) {
+    --columm-count: 3;
```

Comment:
Correct the typo in the custom property name to `--column-count` for consistency and accuracy.